cmd/util/cmd/execution-state-extract/execution_state_extract.go

```diff
@@ -90,6 +91,7 @@ func extractExecutionState(
 		migrations = []ledger.Migration{
```

Comment:
as pointed out previously, creating the migration list unconditionally meant that running migration for non sandbox / benchnet chain will fail.  maybe create different migration list based on the chain rather returning error in addMissingKeysAndUpgradeContractsMigration, line 42.